andela/travel_tool_back
# # 161978026 Rectify travel readiness percentage completion calculation

#### What does this PR do?

- fix travel readiness percentage completion calculation

#### Description of Task to be completed?

- include attribute Id when querying requests table so that percentage completion calculation method is accurate

- This PR also fixes the document name bug when creating or renaming a document.

#### How should this be manually tested?

- Clone the repo - git clone https://github.com/andela/travel_tool_back.git
- Enter into the project directory - cd travel_tool_back
- Checkout the branch `fx-readiness-percentage-completion-161978026`
- Install dependencies - yarn install
- Rollback and run database migrations - yarn db:rollmigrate
- Start the server - make start
- Navigate to travela in your browser and login
- Create a new request  whose destination matches your location and have it approved. You need to assign yourself role of manager - 53019 to be able to do this.
- On the Requests table, Click on the ellipsis to the right and choose `Travel Checklist`. Go ahead to upload a document
- Navigate to the dashboard page to view the Travel Readiness Card. Your travel Readiness percentage should be updated based on the checklist items submitted 

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[161978026](https://www.pivotaltracker.com/story/show/161978026)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/37342753/48565672-2553e500-e90a-11e8-96b0-1542dc7e5088.png)

#### Questions:
